Detour/Source/DetourNavMeshQuery.cpp

```diff
@@ -481,9 +481,9 @@ dtStatus dtNavMeshQuery::findRandomPointAroundCircle(dtPolyRef startRef, const f
 	dtRandomPointInConvexPoly(verts, randomPoly->vertCount, areas, s, t, pt);
 	
 	float h = 0.0f;
-	dtStatus stat = getPolyHeight(randomPolyRef, pt, &h);
+	status = getPolyHeight(randomPolyRef, pt, &h);
```

Comment:
Wouldn't this be overwriting any value set on line 429?  Maybe it should be `status |= getPolyHeight(randomPolyRef, pt, &h);`?